frontend/awx/resources/inventories/InventoryPage/InventoryJobTemplates.tsx

```diff
@@ -0,0 +1,8 @@
+import { useParams } from 'react-router-dom';
+import { awxAPI } from '../../../common/api/awx-utils';
+import { TemplatesList } from '../../templates/TemplatesList';
+
+export function InventoryJobTemplates() {
+  const { id = '' } = useParams<{ id: string }>();
+  return <TemplatesList url={awxAPI`/job_templates/`} inventoryId={id} />;
```

Comment:
The row actions and filter categories are a bit different here than what I see in the old UI- was the `TemplatesList` component changed intentionally or was something missed?

Reply:
Thanks for noticing those! I've added the copy button to the row actions. Just double checked to confirm, and the missing filter categories will likely be added as part of Advanced Search, so for now it should be okay without those.